_usage/query_planning.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: Reviewing a Query Plan
+nav_order: 8
+---
+
+## Reviewing a Query Plan
+
+Babelfish version 2.1.0 includes support for functions and parameters that allow you to review estimated and actual (executed) query plans. This allows you to use the TDS port to identify and refine slow-performing queries. The functions can display an query plan for:
+
+- `SELECT`, `INSERT`, `UPDATE`, and `DELETE` statements
+- Nested procedures, multiple statements in a batch
+- Variable declarations (for example, `DECLARE @t TABLE (a int, b int);`)
+
+The functions *do not* support query plan retrieval from functions, control flows, and cursors.
```

Comment:
this line has 'functions' twice. confusing

Reply:
Now reads: The SET statements *do not* support query plan retrieval from functions, control flows, and cursors.

Reply:
Now reads: The SET statements *do not* support query plan retrieval from functions, control flows, and cursors.